src/utilities/createRouter.ts

```diff
@@ -1,60 +1,68 @@
+import { readonly } from 'vue'
 import { Resolved, Route, RouteMethods, Routes } from '@/types'
-import { createRouteMethods, generateRouteRegexPattern, resolveRoutes, routeParamsAreValid } from '@/utilities'
+import { createRouteMethods, resolveRoutes } from '@/utilities'
+import { resolveRoutesRegex } from '@/utilities/resolveRoutesRegex'
+import { routeMatch } from '@/utilities/routeMatch'
```

Comment:
nit: we already have an import from `@/utilities`, this could just be combined?

Reply:
It could, don't really feel strongly about that personally